.goreleaser.yml

```diff
@@ -6,29 +6,65 @@ release:
 before:
   hooks:
     - go mod tidy
+
 builds:
-  - binary: bin/gh
-    main: ./cmd/gh
-    ldflags:
-      - -s -w -X github.com/cli/cli/command.Version={{.Version}} -X github.com/cli/cli/command.BuildDate={{time "2006-01-02"}}
-      - -X github.com/cli/cli/context.oauthClientID={{.Env.GH_OAUTH_CLIENT_ID}}
-      - -X github.com/cli/cli/context.oauthClientSecret={{.Env.GH_OAUTH_CLIENT_SECRET}}
-      - -X main.updaterEnabled=cli/cli
-    goos:
-      - linux
-      - darwin
-      - windows
-    goarch:
-      - amd64
+  - <<: &build_defaults
+      binary: bin/gh
+      main: ./cmd/gh
+      ldflags:
+        - -s -w -X github.com/cli/cli/command.Version={{.Version}} -X github.com/cli/cli/command.BuildDate={{time "2006-01-02"}}
+        - -X github.com/cli/cli/context.oauthClientID={{.Env.GH_OAUTH_CLIENT_ID}}
+        - -X github.com/cli/cli/context.oauthClientSecret={{.Env.GH_OAUTH_CLIENT_SECRET}}
+        - -X main.updaterEnabled=cli/cli
+    id: macos
+    goos: [darwin]
+    goarch: [amd64]
+  - <<: *build_defaults
```

Comment:
I'm using arcane YAML trickery here to avoid repeating `binary`, `main`, and `ldflags` fields for each of the builds

Reply:
:grimacing: